README.md

```diff
@@ -8,16 +8,22 @@ Apache SkyWalking Satellite
 [![GitHub stars](https://img.shields.io/github/stars/apache/skywalking.svg?style=for-the-badge&label=Stars&logo=github)](https://github.com/apache/skywalking)
 [![Twitter Follow](https://img.shields.io/twitter/follow/asfskywalking.svg?style=for-the-badge&label=Follow&logo=twitter)](https://twitter.com/AsfSkyWalking)
 
-# Quick Start
-
 # Documentation
-- [The first design of Satellite 0.1.0](https://skywalking.apache.org/blog/2020-11-25-skywalking-satellite-0.1.0-design/)
-- [The project structure](./docs/project_structue.md)
+- [Official documentation](https://skywalking.apache.org/docs/)
+- [Blog](https://skywalking.apache.org/blog/2020-11-25-skywalking-satellite-0.1.0-design/) about the design of Satellite 0.1.0.
+
+NOTICE, SkyWalking Satellite uses [v3 protocols](https://github.com/apache/skywalking/blob/master/docs/en/protocols/README.md). They are incompatible with previous SkyWalking releases before SkyWalking 8.0.
 
 # Download
+Go to the [download page](https://skywalking.apache.org/downloads/) to download all available binaries, including MacOS and Linux.
+If you want to try the latest features or run on the Windows, however, you can compile the latest codes yourself, as the guide below. 
 
 # Compile
-[How to compile the Satellite.](./docs/compile/compile.md)
+As SkyWalking Satellite is using `Makefile`, compiling the project is as easy as executing a command in the root directory of the project.
+```shell script
+make build
+```
+Due to system compatibility problems, some plugins of SkyWalking Satellite cannot be used in Windows system. If you need to compile SkyWalking Satellite on Windows platform, please read [the doc](docs/en/guides/compile/compile.md).
```

Comment:
I can see `Windows` is your makefile, what do you mean ?

Reply:
platform means the supported platform，however，it not works. It only works when append windows  to the build

Reply:
fixed